### ISSUE TITLE
MultiScreenInfo.surfaceDuoInfoModel.isSpanned = false while SurfaceDuoPlatformHandler.getIsSpanned() = true

### ISSUE THREAD

**Issue by kimonneuhoff** [MisterJimson/multi_screen_layout#17]
Hello,
I made this simple example. When I debugged this application everything worked fine. As soon as I wanted to deploy it in release mode an error ("java.lang.AbstractMethodError: abstract method "void androidx.window.sidecar.SidecarInterface$SidecarCallback.onDeviceStateChanged(androidx.window.sidecar.SidecarDeviceState)"") came which I resolved by adding 
`dependencies {
    implementation "androidx.window:window:1.0.0-beta04"
}`
to the build.gradle file. Now the app runs but it seems to not think the device is not Spanned when it is spanned so I always just see the first child from TwoPageLayout.

```
MultiScreenInfo(builder: (info) {
        print(info.surfaceDuoInfoModel.isSpanned);
}); **returns false;**

SurfaceDuoPlatformHandler.getIsSpanned().then((value) => print(value)); **returns true**

Code: https://www.toptal.com/developers/hastebin/isukovusaj.java
or
`import 'dart:async';

import 'package:flutter/material.dart';
import 'package:multi_screen_layout/multi_screen_layout.dart';

void main() {
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({Key? key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Spikeball Counter',
      theme: ThemeData(
        primarySwatch: Colors.blue,
      ),
      home: MultiScreenInfo(builder: (info) {
        print(info.surfaceDuoInfoModel.isSpanned);
        SurfaceDuoPlatformHandler.getIsSpanned().then((value) => print(value));
        return TwoPageLayout(child: const MyHomePage(color: Colors.blue), secondChild: MyHomePage(color: Colors.red.shade400));}),
    );
  }
}

class MyHomePage extends StatefulWidget {
  const MyHomePage({Key? key, required this.color}) : super(key: key);
  final Color color;

  @override
  State<MyHomePage> createState() => _MyHomePageState();
}

class _MyHomePageState extends State<MyHomePage> {
  int _counter = 0;
  bool _winner = false;
  Color bg = Colors.white;

  @override
  void initState() {
    super.initState();
    bg = widget.color;
  }

  void _incrementCounter() {
    setState(() {
      _counter++;
    });
  }

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      body: GestureDetector(
          onTap: () {
            if(_winner) return;
            _incrementCounter();
            if(_counter == 25) {
              _winner = true;
              Timer.periodic(const Duration(milliseconds: 500), (Timer timer) {
                  if (_counter != 25) {
                    setState(() {
                      _winner = false;
                      timer.cancel();
                      bg = widget.color;
                    });
                  } else {
                    setState(() {
                      bg = (bg == widget.color ? Colors.white : widget.color);
                    });
                  }
                },
              );
            }
          },
          child: Container(color: bg, child: Center(
        child: Column(
          mainAxisAlignment: MainAxisAlignment.center,
          children: <Widget>[
            Text(
              'Team ' + (widget.color == Colors.red.shade400 ? 'Rot' : 'Blau'), style: TextStyle(fontSize: 20, color: (widget.color == Colors.red.shade400 ? Colors.red[900] : Colors.blue[900])),
            ),
            Text(
              '$_counter',
              style: TextStyle(fontSize: 280, color: (widget.color == Colors.red.shade400 ? Colors.red[900] : Colors.blue[900])),
            ),
          ],
        )),
      )),
      floatingActionButton: Row(mainAxisAlignment: MainAxisAlignment.end, children: [
        IconButton(onPressed: () {setState(() {
          if(_counter == 0) return;
        _winner = false;
        _counter -= 1;
      });}, icon: Icon(Icons.exposure_minus_1_rounded)),
        IconButton(onPressed: () {setState(() {
        _winner = false;
        _counter = 0;
      });}, icon: Icon(Icons.refresh_rounded))],),
    );
  }
}
```

**Comment by MisterJimson** [comment]
Hello! Which version of the library are you using?

**Comment by kimonneuhoff** [comment]
The latest (3.0.0)

**Comment by kimonneuhoff** [comment]
Hello, is there any Update on this? :o

**Comment by MisterJimson** [comment]
Hey! I plan to look into this soon. Probably early next week.

**Comment by MisterJimson** [comment]
I'm able to run your code no problem and it displays TwoPageLayout correctly. I'm running it on a Surface Duo 2, are you using a Surface Duo 1, 2 or the emulator?

**Comment by MisterJimson** [comment]
Ah I was able to reproduce it with adding the specific version of the window manager lib.

Can you replace:
```
dependencies {
    implementation "androidx.window:window:1.0.0-beta04"
}
```

with 
```
dependencies {
    implementation "androidx.window:window:1.0.0-alpha06"
}
```

And test again?

**Comment by kimonneuhoff** [comment]
Unfortunately the error "java.lang.AbstractMethodError: abstract method "void androidx.window.sidecar.SidecarInterface$SidecarCallback.onDeviceStateChanged(androidx.window.sidecar.SidecarDeviceState)" is thrown again after replacing this

**Comment by kimonneuhoff** [comment]
I also tried it with the latest Version https://developer.android.com/jetpack/androidx/releases/window but It had the same result as using beta4 where 
MultiScreenInfo.surfaceDuoInfoModel.isSpanned returns false while SurfaceDuoPlatformHandler.getIsSpanned() returns true

**Comment by MisterJimson** [comment]
I just published version 3.1 which updates the Window Manager lib to newest, it required some code changes.

These are the deps that would need to be added after upgrading to 3.1.

```
dependencies { 
    implementation "androidx.window:window:1.0.0-rc01"
    implementation 'androidx.window:window-java:1.0.0-rc01'
}
```

**Comment by kimonneuhoff** [comment]
It's working!! Thank you. ☺
